src/Commands/Shooter/Shoot.h

```diff
@@ -16,6 +16,8 @@ class Shoot: public Command {
 
 private:
     const float OPTIMAL_RPM = -3300.0f;
+    const float THROTTLE = 500.0f;
     const float IDLE = 1000.0f;
     const float START_MULTIPLIER = 1.25f;
+    const double DEADBAND = 0.01;

```

Comment:
This deadband is probably too small, might need to be somewhere between .02 and .05. This will differ per controller.